test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
It seems to me that you accidentaly commited this script, oder?

Reply:
good catch